contracts/bridge/x/v4/CoreV4.sol

```diff
@@ -0,0 +1,278 @@
+// Copyright (c) Immutable Pty Ltd 2018 - 2024
+// SPDX-License-Identifier: MIT
+//
+// This contract is an interface for the StarkEx Core contract v4 version.
+// It is used to interact with the StarkEx Core contract from the Registration contract.
+// The Core contract is used to register and withdraw users and assets from the StarkEx system.
+//
+// This file was generated using the abi-to-sol tool.
+// the StarkEx contract ABI that was provided by StarkWare in our slack connect channel (#ext-immutable-starkware)
```

Comment:
Let's just say 
"the StarkEx contract ABI that was provided by StarkWare via slack."

We shouldn't put the internal details of the slack channel and a link to the slack item in here.